framework/src/controller/controller.js

```diff
@@ -127,6 +130,7 @@ class Controller {
 			'lisk',
 			['ready'],
 			{
+				isLiskReady: () => this.liskReady,
```

Comment:
Why we don't need this action? We must not create actions preferably if the it can be sorted through events. And `lisk:ready` event is the right solution for it.

Reply:
We need this when reloading the module